storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -93,6 +95,34 @@ export const WithDescription: Story = {
   ),
 }
 
+export const WithHint: Story = {
+  args: { optional: false, hint: 'verplicht' },
+  render: ({ invalid, legend, hint, optional }) => (
+    <FieldSet invalid={invalid} legend={legend} optional={optional} hint={hint}>
+      <Field className="ams-mb--sm">
+        <Label htmlFor="input-a1">Voornaam</Label>
+        {invalid && <ErrorMessage id="error-a1">Vul uw voornaam in.</ErrorMessage>}
```

Comment:
You shouldn't use the same `id` (`error-a1` for example) more than once on a page.

Reply:
Fixed